packages/devtools_app/lib/main.dart

```diff
@@ -98,8 +98,8 @@ String _generateSurveyUrl() {
 
   final fromValue = uri.fragment ?? '';
 
-  // TODO(kenz): get actual value of whether user is internal or external.
-  const internalValue = 'false';
+  final response = await HttpRequest.request('/api/getUserLdap');
```

Comment:
Suggestion: call this `ldapResponse` to be clear about what the contents of the response are.

Reply:
done.